colorcoder/ColorCode.java

```diff
@@ -0,0 +1,23 @@
+package colorcoder;
+
+public class ColorCode {
+	static int GetColorCodeFromColor(MajorColor major, MinorColor minor) {
+		int Colorcode = 0;
+		Colorcode = major.getIndex() * 1 + minor.getIndex() + 1;
+		switch (major.getIndex()) {
+		case 1:
+			Colorcode = Colorcode + 4;
```

Comment:
4, 8,12,16 are magic numbers. These are bound to change in case the number of minor colours increase or decrease.
Need a better way to safeguard such changes.

Reply:
Okay bro I will change the logic and commit.